shoppingcart/server/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/temporalio/samples-go/shoppingcart"
+	enumspb "go.temporal.io/api/enums/v1"
+	"go.temporal.io/sdk/client"
+	"log"
+	"net/http"
+	"sort"
+)
+
+var (
+	cartState      = make(map[string]int)
+	workflowClient client.Client
+	itemCosts      = map[string]int{
+		"apple":      2,
+		"banana":     1,
+		"watermelon": 5,
+		"television": 1000,
+		"house":      10000000,
+		"car":        50000,
+		"binder":     10,
```

Comment:
I wonder whether these should be cents, in the interests of making the sample more like real code.